AppCenterDistribute/AppCenterDistribute/MSDistribute.m

```diff
@@ -225,6 +235,31 @@ + (void)setDelegate:(id<MSDistributeDelegate>)delegate {
   [[MSDistribute sharedInstance] setDelegate:delegate];
 }
 
+#pragma mark - In-app updates
+
++ (void)setUpdateTrack:(MSUpdateTrack)updateTrack {
+  @synchronized(self) {
```

Comment:
Move the implementation to instance method.
Then you don't need the backing variable.